go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
+github.com/bep/clocks v0.5.0/go.mod h1:SUq3q+OOq41y2lRQqH5fsOoxN8GbxSiT6jvoVVLCVhU=
```

Comment:
Review the specific versions of the libraries being used. For example, some libraries like `github.com/dgrijalva/jwt-go` have known vulnerabilities and are not maintained anymore. Consider switching to the maintained fork (e.g., `github.com/golang-jwt/jwt`).